package.json

```diff
@@ -2,7 +2,7 @@
   "name": "purs-loader",
   "version": "3.1.2",
   "description": "A webpack loader for PureScript.",
-  "main": "lib/index.js",
+  "main": "src/index.js",
```

Comment:
The  reason this is `lib` is because the loader src is transpiled by babel to the lib directory when it is published. We will have to keep it as lib I think.

Reply:
Ahh - fair enough. I just pointed to my fork on github and this was needed.